views/pages/terms/index.ejs

```diff
@@ -2,9 +2,20 @@
     <div class="text-center bold">Terms</div>
 </div>
 
-<div class="card bg-primary" id="body">
-    <div class="text-center bold" contenteditable="true" id="Term1">Term 1</div>
+<div class="card bg-primary" id="Term1">
+    <div class="text-center bold" contenteditable="true" id="userTerm1">Term 1 </div> 
+    <div align="right">
```

Comment:
Align is deprecated, should use text-align

Inline styles should be style="text-align:right;"

Reply:
I have now made the changes that you have suggested.